pkg/kv/kvserver/batcheval/cmd_add_sstable_test.go

```diff
@@ -324,6 +324,33 @@ func TestEvalAddSSTable(t *testing.T) {
 			sst:        kvs{pointKV("a", 3, "sst")},
 			expectErr:  &roachpb.WriteTooOldError{},
 		},
+		"DisallowConflicts returns WriteTooOldError at existing range tombstone": {
+			noConflict: true,
+			data:       kvs{rangeKV("a", "c", 3, "")},
+			sst:        kvs{pointKV("b", 3, "sst")},
+			expectErr:  &roachpb.WriteTooOldError{},
+		},
+		// Regression tests for https://github.com/cockroachdb/cockroach/issues/93968.
+		"DisallowConflicts WriteTooOldError straddling at existing range tombstone": {
+			noConflict: true,
+			data:       kvs{rangeKV("b", "z", 3, "")},
+			sst:        kvs{pointKV("a", 3, "sst"), pointKV("c", 3, "sst")},
+			expectErr:  &roachpb.WriteTooOldError{},
+		},
+		"DisallowConflicts WriteTooOldError straddling multiple existing range tombstones": {
+			noConflict: true,
+			data:       kvs{rangeKV("b", "d", 3, ""), rangeKV("d", "f", 3, "")},
```

Comment:
For what's worth, both of these will get defragmented together at read time as the prefix/values match

Reply:
Whoops, was supposed to be a gap there, thanks -- messed up while I was simplifying the test case.